Makefile

```diff
@@ -88,6 +91,15 @@ envtest:
 lint:
 	@golangci-lint run --timeout 15m
 
+checkov: _work/venv/.stamp
```

Comment:
Do we really need sphinx packages that are installed by _work/venv/.stamp target?

Reply:
No, just I thought it's better to use what already exists then just putting "python3 -m venv ..."